SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriverRPC.h

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Interop/RPCs/RPCService.h"
+#include "SpatialView/EntityComponentId.h"
+#include "SpatialNetDriverRPC.generated.h"
+
+namespace SpatialGDK
+{
+class SpatialEventTracer;
+}
+
+struct FRPCPayload
```

Comment:
Why dupe this functionality when it's available in `RPCPayload` ?

Reply:
RPCPayload is a one-size-fits-all that got a bit out of hand. I want to give us the opportunity for a fresh start, and also have different payload types to make sure the infrastructure works for all of them.